app/models/payment_confirmation.rb

```diff
@@ -36,8 +39,8 @@ def ingest
 
   private
 
-  def payment_date_timestamp
-    Date.today.next_occurring(:friday).to_time.to_i
+  def payment_date
```

Comment:
If we're renaming the column maybe we should also rename this method to `scheduled_payment_date`?